config.lua

```diff
@@ -1,8 +1,8 @@
 Config = {}
 
 -- **** IMPORTANT ****
--- UseTarget should only be set to true when using qb-target
-Config.UseTarget = GetConvar('UseTarget', 'false') == 'true'
+Config.UseTarget = true
```

Comment:
It was defaulting to false before, now it defaults to true. Why the change?

Reply:
so do i need to set it false or add the Convar back ?

Reply:
I think this is fine actually. It's a change to use target by default.